src/lib/poracleMessage/commands/enable.js

```diff
@@ -17,12 +17,12 @@ exports.run = async (client, msg, args, options) => {
 		// Make list of ids
 		const mentions = msg.getMentions()
 		const targets = mentions.map((x) => x.id)
-		targets.push(args.filter((x) => parseInt(x, 10)))
+		targets.push(...(args.filter((x) => parseInt(x, 10))))
```

Comment:
This is absolutely a bug (this change mirrors disable)

Reply:
Do you mean you agree that it was a bug or the fix is a bug?

Reply:
I agree that your change fixes the bug! Sorry if I was unclear

Reply:
Ah okay, no problem!